libs/compute_api/src/responses.rs

```diff
@@ -135,26 +135,6 @@ pub struct CatalogObjects {
     pub databases: Vec<Database>,
 }
 
-/// Response of the `/computes/{compute_id}/spec` control-plane API.
-/// This is not actually a compute API response, so consider moving
-/// to a different place.
-#[derive(Deserialize, Debug)]
-pub struct ControlPlaneSpecResponse {
```

Comment:
Why not API? This comment is more about moving into a different file or something

compute_api is common dir for compute interface, we also share code owners on it with cplane, so they know about any changes. -1 for this PR

Reply:
Maybe you can refresh comments and/or add readme clarifying this and/or put it into a separate file, but this belongs to `compute_api` as it's sort of bi-directional API with cplane

Reply:
Not sure I follow. The compute_api crate from my understanding is for response and request types related to the compute http API. We should probably create a separate `cplane_api` crate which is composed of request and response types of the cplane compute API. What do you think?

Reply:
+1 on a separate crate. Or at least a separate file within this crate, or just separate with comments. 

To clarify there are two different APIs here:
1. compute ->cplane
2. cplane -> compute

All the other request/responses in the crate belong to the cplane->compute API, but this one is compute->cplane.

Reply:
Both ways work for me; just set codeowners right, so for both files/crate, cplane+compute are owners, and all API changes are visible. In the current state, PR makes any changes invisible to cplane folks

Reply:
Cool. I will put a few minutes into this at some point later this week after some other work merges!